users/vieting/experiments/switchboard/hybrid/feat/baseline_args.py

```diff
@@ -23,21 +23,26 @@
     get_funcs_jingjing,
 )
 
+from .specaug_sorted import ( 
+    specaug_layer_sorted,
+    get_funcs_specaug,
```

Comment:
```suggestion
    get_funcs_sorted,
```
Seems more consistent to the remaining naming.

Reply:
Please also change the remaining occurrences.